packages/lib/src/test/jsonld.test.ts

```diff
@@ -72,6 +76,17 @@ describe('check', () => {
     const result = await check(vc);
     expect(result.ok).toBeTruthy();
   });
+
+  it('should pass json with no mapped properties', async () => {
+    const result = await check(docOnlyId);
+    expect(result.ok).toBeTruthy();
+    const result2 = await check(docNoMappedProp);
+    expect(result2.ok).toBeTruthy();
+    const result3 = await check(docOnlyIdOnePropNoMap);
+    expect(result3.ok).toBeFalsy();
+    const result4 = await check(docOnlyIdOneProp);
+    expect(result4.ok).toBeTruthy();
+  });
 });
 
```

Comment:
also can we add a test to show that it handles a doc with an emoji as a prop (see orie's example)